src/LinkDotNet.Blog.Domain/BlogPost.cs

```diff
@@ -35,6 +37,18 @@ private BlogPost()
 
     public string TagsAsString => Tags is null ? string.Empty : string.Join(", ", Tags);
 
+    [NotMapped]
+    public string SearchEngineFriendlyUrl => GenerateSearchEngineFriendlyUrl();
+
+    private string GenerateSearchEngineFriendlyUrl()
+    {
+        string SearchEngineFriendlyTitle = Title
```

Comment:
That will not cover all the cases.
For example if my title has "/" inside, this will fail.

So a wide test net is needed.